index.js

```diff
@@ -46,30 +46,39 @@ cs.get.rgb = function (string) {
 		return null;
 	}
 
-	var abbr = /^#([a-fA-F0-9]{3})$/;
-	var hex = /^#([a-fA-F0-9]{6})$/;
+	var abbr = /^#([a-f0-9]{3})([a-f0-9])?$/i;
```

Comment:
There isn't a reason to have a second clause on `abbr` - only `hex`. The reason I asked for it on `hex` was because it would match a 7 character hex string, which isn't correct (we want either 6 or 8, but not 7). You can change this back to just `/^#([a-f0-9]{3,4})$/i` :)

L50 (`var hex = ...`) looks great, though :)